src/services/assetService.ts

```diff
@@ -200,15 +200,9 @@ export class AssetService {
         const objectArray = await this.getTFRecordObjectArrays(asset);
         const regions: IRegion[] = [];
         const tags: string[] = [];
-        let tagPos = 0;
 
         // Add Regions from TFRecord in Regions
         for (let index = 0; index < objectArray.textArray.length; index++) {
-            tagPos = tags.findIndex((tag) => tag === objectArray.textArray[index]);
```

Comment:
why are we removing this?

Reply:
If you look at the code below you see this is not needed anymore.